Library/Homebrew/cmd/uses.rb

```diff
@@ -108,4 +118,80 @@ def uses
     puts Formatter.columns(uses.map(&:full_name).sort)
     odie "Missing formulae should not have dependents!" if used_formulae_missing
   end
+
+  def intersection_of_dependents(used_formulae, recursive = false)
+    if @use_runtime_dependents
+      return used_formulae.map(&:runtime_installed_formula_dependents)
+                          .reduce(&:&)
```

Comment:
Could you explain what this does? I'm just curious.

Reply:
I'm not sure, I _think_ it removes duplicate entries; I copied it from the lines above